include/ppx/application.h

```diff
@@ -327,23 +327,23 @@ struct ApplicationSettings
     // Default values for standard knobs
     struct StandardKnobsDefaultValue
     {
-        std::vector<std::string> assetsPaths     = {};
-        std::vector<std::string> configJsonPaths = {};
-        bool                     deterministic   = false;
-        bool                     enableMetrics   = false;
-        uint64_t                 frameCount      = UINT64_MAX;
-        int                      gpuIndex        = INT_MAX;
+        const std::vector<std::string> assetsPaths     = {};
+        const std::vector<std::string> configJsonPaths = {};
+        bool                           deterministic   = false;
+        bool                           enableMetrics   = false;
+        uint64_t                       frameCount      = 0;
+        int                            gpuIndex        = 0;
 #if !defined(PPX_LINUX_HEADLESS)
         bool headless = false;
 #endif
         bool                listGpus              = false;
-        std::string         metricsFilename       = std::filesystem::current_path().u8string();
+        std::string         metricsFilename       = "report_@.json";
```

Comment:
i have a crash in `std::filesystem::create_directories` with `report_@.json` as metricsFilename on windows

Reply:
Can you tell me which dir you were executing this from?

Reply:
i am running from vs2022, the `Working Directory` is set to `$(ProjectDir)`